tools/monorepo-utils/src/workflow-profiler/lib/math.ts

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
I'm ambivalent regarding this module. 

On the one hand, this is a (tiny) descriptive stats library and should probably be replaced by something specialized, like [mathjs](https://www.npmjs.com/package/mathjs). 

On the other hand, adding more and more dependencies for basic stuff can obviously get out of hand, and sticking to pragmatism here is a very valid perspective also. 

A middle ground could be moving this to `packages` as a proper ... package. 

Thoughts? 🤔